src/utils/breaches.js

```diff
@@ -24,10 +25,25 @@ import { filterBreachDataTypes } from './breachResolution.js'
  * @param {*} allBreaches
  * @returns {Promise<AllEmailsAndBreaches>}
  */
-async function getAllEmailsAndBreaches (user, allBreaches) {
-  const monitoredEmails = await getUserEmails(user.id)
+async function getAllEmailsAndBreaches(user, allBreaches) {
+  
+  // @ts-ignore: function will be deprecated
   const verifiedEmails = []
+  // @ts-ignore: function will be deprecated
   const unverifiedEmails = []
+  
+  if (!user) {
+    console.error("subscriber cannot be undefined");
+    // @ts-ignore: function will be deprecated
+    return { verifiedEmails, unverifiedEmails };
+  }
+  if (!allBreaches || allBreaches.length === 0) {
+    console.error("all breaches object cannot be empty");
```

Comment:
Not sure if this is better or clearer, but now matches the argument name.
```suggestion
    console.error("allBreaches object cannot be empty");
```

Reply:
Could you do Sentry `captureError()` here as well? Then it'll show up when we're doing triage.